### ISSUE TITLE
PowerShell deployment failing, but doesn't say why

### ISSUE THREAD

**Issue by ameltzerMSFT** [microsoft/winget-cli-restsource#168]
### Brief description of your issue

This may be a duplicate of #131. I'm trying to run the New-WinGetSource command and it fails with an error:

> New-WinGetSource: Testing found an error with the ARM template or parameter files.
> New-WinGetSource: ARM Template and Parameter testing failed
> (dump of key/value pairs)

### Steps to reproduce

Example output: 
```shell
PS C:\src\winget-cli-restsource\src> new-wingetsource -name "xxx" -ResourceGroup xxx -Region xxx -SubscriptionName "xxx"
True
New-WinGetSource: Testing found an error with the ARM template or parameter files.
New-WinGetSource: ARM Template and Parameter testing failed


Name                           Value
----                           -----
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\applicationinsights.json
ObjectName                     demo
ObjectType                     AppInsight
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\applicationinsights.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\keyvault.json
ObjectName                     demo
ObjectType                     Keyvault
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\keyvault.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\storageaccount.json
ObjectName                     demo
ObjectType                     StorageAccount
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\storageaccount.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\asp.json
ObjectName                     demo
ObjectType                     asp
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\asp.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\cosmosdb.json
ObjectName                     demo
ObjectType                     CosmosDBAccount
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\cosmosdb.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\cosmosdb-sql.json
ObjectName                     WinGet
ObjectType                     CosmosDBDatabase
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\cosmosdb-sql.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\cosmosdb-sql-container.json
ObjectName                     Manifests
ObjectType                     CosmosDBContainer
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\cosmosdb-sql-container.json
Parameters                     {$Schema, contentVersion, Parameters}
ParameterPath                  C:\src\winget-cli-restsource\src\Parameters\azurefunction.json
ObjectName                     demo
ObjectType                     Function
Error
TemplatePath                   C:\src\winget-cli-restsource\Tools\PowershellModule\src\Library\ARMTemplate\azurefunction.json
```

### Expected behavior

Clearly explain what the failure reason is

### Actual behavior

No clear explanation

### Environment

```shell
PS C:\src\winget-cli-restsource\src> winget --info
Windows Package Manager v1.3.2091
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1889
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2091.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by RDMacLachlan** [comment]
Problem has been identified. Working on implementing the fix now.

**Comment by RDMacLachlan** [comment]
The fix for this issue is in PR #169
